decidim-age_action_authorization/README.md

```diff
@@ -44,13 +44,11 @@ Create a dummy app in your application (not in the module):
 cd ..
 bin/rails decidim:generate_external_test_app
 cd spec/decidim_dummy_app/
-bundle exec rails decidim_age_action_authorization:install:migrations
```

Comment:
is the `decidim:generate_external_test_app` already installing this module's migrations?